student/models.py

```diff
@@ -59,6 +59,7 @@ class year, major, friends, etc. An object is only created for the
     first_name = models.CharField(max_length=255, default='', null=True)
     last_name = models.CharField(max_length=255, default='', null=True)
     disabilities = models.NullBooleanField(null=True, default=False)
+    favorite_num = models.IntegerField(blank=True, null=True)
```

Comment:
You can also specify a default number with integer fields as well :) You can work without it but it is sometimes a good and necessary design choice to add a default value to new fields! 
`favorite_num = models.IntegerField(blank=True, null=True, default=0)`